brion/spikeReport.h

```diff
@@ -279,6 +287,12 @@ class SpikeReport : public boost::noncopyable
      */
     BRION_API void write( const Spikes& spikes );
 
+    /**
+     * @return Whether the report supports seek to t < getCurrentTime() or not.
+     * @version 2.0
+     */
+    BRION_API bool supportsBackwardSeek() const;
+
```

Comment:
Seek returns a future and throws if backward seek is not supported. It would be better to know in advance if the feature is available or not.